sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareClientBuilder.java

```diff
@@ -652,7 +652,8 @@ public ShareClientBuilder allowTrailingDot(boolean allowTrailingDot) {
 
     /**
      * Sets the {@link ShareTokenIntent} that specifies whether there is intent for a file to be backed up.
-     * This is currently required when using {@link TokenCredential}.
+     * Possible values are: {@link ShareTokenIntent#BACKUP}
```

Comment:
Why do we need this "possible values" section? Seems like one more thing to fall out of date and they can easily determine possible values as this enum is custom made for this.

Same for all the other instances.